src/stravalib/model.py

```diff
@@ -400,21 +316,24 @@ def check_sport_type(cls, values: dict[str, Any]) -> dict[str, Any]:
         dict
             A dictionary with a validated sport type value assigned.
         """
-        v = values["__root__"]
-        if v not in get_args(SportType.__fields__["__root__"].type_):
+        # Do we want to deprecated SportType? os because it's still returned
```

Comment:
I don't understand this. Why should `SportType` be deprecated? According to Strava, this is the type that should be used.

Reply:
apologies - this was my mistake. You are correct. i mixed up sport and activity type.  i'll remove this comment